src/components/page-layout/split-page.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
Tucking this component in here for now. This branch will merge into the PageLayouts #309 branch and I'll follow up with any necessary documentation in that PR.